roles/ec2_provisioning/defaults/main.yml

```diff
@@ -1,11 +1,12 @@
-keypair: django-keys
-security_group: launch-wizard-1
+keypair: ec2
+security_group: ec-security-group
 instance_type: t2.micro
 ami: ami-d732f0b7
-vpc_subnet_id: subnet-e4142480
+vpc_subnet_id: subnet-c38b82a7
 region: us-west-2
-instance_tags: {"Name": "ansible-built"}
+instance_tags: {"Name": "django-imager"}
```

Comment:
Here's another place where you could use a variable, rather than a project-specific name.